engines/standard_tasks/app/models/standard_tasks/paper_reviewer_task.rb

```diff
@@ -6,6 +6,18 @@ def self.permitted_attributes
 
     register_task default_title: "Assign Reviewers", default_role: "editor"
 
+    include Invitable
+
+    def invitation_invited(invitation)
+      PaperReviewerMailer.delay.notify_invited({
+        invitation_id: invitation.id
+      })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.